src/preact/base-element.js

```diff
@@ -610,6 +629,107 @@ export class PreactBaseElement extends AMP.BaseElement {
     }
     return this.defaultProps_[prop];
   }
+
+  /**
+   * Returns reference to upgraded imperative API object, as in React's
+   * useImperativeHandle.
```

Comment:
Please also add notes for `getApi()` and `api()` when one is recommended over the other.